apps/src/lab2/views/components/PredictSummary.tsx

```diff
@@ -17,18 +21,53 @@ const PredictSummary: React.FunctionComponent = () => {
     `viewAs=${ViewType.Instructor}`
   );
   const summaryUrl = document.location.pathname + SUMMARY_PATH + params;
+  const currentSectionId = useSelector(
+    (state: {teacherSections: {selectedSectionId: number}}) =>
+      state.teacherSections.selectedSectionId
+  );
+  const currentLevelId = useAppSelector(state => getCurrentLevel(state)?.id);
+  const [responseCount, setResponseCount] = React.useState<number | null>(null);
+  const [numStudents, setNumStudents] = React.useState<number | null>(null);
```

Comment:
super minor - but just for consistency, separate import for `useState` since you're already importing `useEffect`?

Reply:
yep that was just a miss on my part!